src/Place/DuplicatePlace/ImportDuplicatePlacesProcessor.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\DuplicatePlace;
```

Comment:
I would avoid having 2 namespaces for the same thing. We have a `Canonical` namespace and now also a `DuplicatePlace`. I would put everything in the `Canonical` namespace for now and in a next pull request you can rename that to `Duplicates`